src/Graphics/UI.h

```diff
@@ -75,7 +75,7 @@ class UI
     int _y = 0;
     int _xOffset = 0;
     int _yOffset = 0;
-    Texture* _texture = 0;
+    mutable Texture* _texture = 0;
```

Comment:
It's a specific to inherited classes but not to UI class.
And there can be a better solution that affects only classes like BigCounter.
I'll spend some more time on that.